tac/agents/controller/agent.py

```diff
@@ -178,7 +178,7 @@ def start(self) -> None:
         """
         try:
             super().start()
-            logger.debug("[{}]: Starting myself...".format(self.name))
+            logger.debug("[{}]: Stopping myself...".format(self.name))
```

Comment:
@marcofavorito why did you update that..seems wrong

Reply:
That line is executed when the agent exits from the main loop (in `super().start()`).